### PR TITLE
docs: add RFC for code owners

### PR DESCRIPTION
**Pull request by JoshuaLicense**
## Description

Add RFC for code owners

### CODE REVIEW

**Review by sdh100shaun — DISMISSED** [review]

File: docs/rfc/rfc-003-code-owners.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+### Time Investment
+
+Allocate roughly 5% of a developer's capacity each sprint (equivalent to one half-day per sprint) for code owner tasks. This can be done once a month, so a full day every two sprints to work on these tasks. _Sprints should be planned with this in mind; sprint work will take precedence over code-owner tasks._
+
+### Coordination
+
+The scrum events will be used to coordinate the code owner's activities:
+
+-   **Sprint Planning**: Allocate time for code owner tasks.
```
Comment (hobbyhacker0) [inline docs/rfc/rfc-003-code-owners.md:41]: Is the 5% implicit? So:
...time for code owner tasks_, beyond the 5%, if any._

File: docs/rfc/rfc-003-code-owners.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+-   **Daily Stand-ups**: Briefly discuss ongoing code owner activities.
+-   **Retrospective**: Discuss the effectiveness of the code owner's activities and identify improvements.
```
Comment (hobbyhacker0) [inline docs/rfc/rfc-003-code-owners.md:43]: of code owner activity

File: docs/rfc/rfc-003-code-owners.md (hunk 1 above)
Comment (JoshuaLicense) [reply docs/rfc/rfc-003-code-owners.md:41]: The 5% can be "banked" (so a full day per month perhaps if that works better for people).

The intention was that larger pieces of work that take longer than a half-day (5%) should be normal tickets that can be prioritised by the team/business.

**Review by fibble — COMMENTED** [review]
Will we be adopting the CODEOWNERS file spec as part of this? Sorry if that is implied, just wondering if owners names/handles will be defined in that standardized way or somehwere else? Should we specify this in the RFC?

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-file-location

**Comment by JoshuaLicense** [comment]
> Will we be adopting the CODEOWNERS file spec as part of this? Sorry if that is implied, just wondering if owners names/handles will be defined in that standardized way or somehwere else? Should we specify this in the RFC?
> 
> https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-file-location

Yeah, sorry that isn't clear. GitHub Codeowners will be used for this.

**Review by fibble — APPROVED** [review]

**Pull request merged**